cmd2/cmd2.py

```diff
@@ -575,6 +580,15 @@ def register_command_set(self, cmdset: CommandSet) -> None:
         if type(cmdset) in existing_commandset_types:
             raise CommandSetRegistrationError('CommandSet ' + type(cmdset).__name__ + ' is already installed')
 
+        if self.always_prefix_settables:
+            if len(cmdset.settable_prefix.strip()) == 0:
+                raise CommandSetRegistrationError('CommandSet settable prefix must not be empty')
+        else:
+            all_settables = self.settables
+            for key in cmdset.settables.keys():
```

Comment:
Can this be combined into the following?
`if any(key in all_settables for key in cmdset.settables.keys()):`

Reply:
It can - but it makes 'key' inaccessible to the exception being raised.